notion-distcopy/src/main/scala/com/ambiata/notion/distcopy/DistCopyJob.scala

```diff
@@ -45,11 +47,15 @@ object DistCopyJob {
         job.setJobName(ctx.id.value)
         job.setInputFormatClass(classOf[DistCopyInputFormat])
         job.getConfiguration.setBoolean("mapreduce.map.speculative", false)
-        job.getConfiguration.setInt("mapreduce.map.maxattempts", 1)
+        job.getConfiguration.setInt("mapreduce.map.maxattempts", if(conf.parameters.crossValidate) 1 else 3)
```

Comment:
@markhibberd I changed this to up the mapper retry if cross validate is set to false